apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -4,7 +4,7 @@ export function imageFromMedia(alt, url) {
   return { alt, src: url };
 }
 
-function getFooter(siteSettings) {
+function getFooter(siteSettings, variant) {
```

Comment:
Minor but taking a `page` is probably better i.e. settings and page in question.
```suggestion
function getFooter(siteSettings, page) {
```

Or `function getFooter(siteSettings, { variant })`;

Reply:
@kilemensi This won't work as we're fetching the variant from `globals` not the page itself.